spec/features/generic_announcements_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe 'Generic announcements', type: :feature do
```

Comment:
On second thought this doesn't describe a workflow. It just tests that what is rendered in the view meets some criteria, given some data. Maybe I should make this a controller spec instead?